### DISCUSSION TITLE
Tip and tricks

### DISCUSSION THREAD

**Discussion by 1007john** [community/community#60143] [Other Feature Feedback, Questions, & Ideas] [answered]
How to find great tip and tricks on your search.

**Comment by GhostOf0days** [comment]
[GitHub Protips: Tips, Tricks, Hacks, and Secrets from Lee Reilly](https://github.blog/2020-04-09-github-protips-tips-tricks-hacks-and-secrets-from-lee-reilly/)
[Git Tips](https://github.com/git-tips/tips)
[Learn Git Branching](https://learngitbranching.js.org/?locale=en_US)
[GitHub Quickstart: Hello World](https://docs.github.com/en/get-started/quickstart/hello-world)
[The Beginner's Guide to Git & GitHub](https://www.freecodecamp.org/news/the-beginners-guide-to-git-github/)

**Comment by queenofcorgis** [comment, marked as answer]
Welcome to the GitHub Community, @1007john , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by 1007john** [reply]
Hi queenofcorgis,
Thanks
